packages/@react-stately/utils/src/number.ts

```diff
@@ -18,20 +18,20 @@ export function clamp(value: number, min: number = -Infinity, max: number = Infi
   return newValue;
 }
 
-export function snapValueToStep(value: number, min: number, max: number, step: number): number {
-  let remainder = ((value - (isNaN(min) ? 0 : min)) % step);
+export function snapValueToStep(value: number, min: number | undefined, max: number | undefined, step: number): number {
+  let remainder = ((value - (isNaN(Number(min)) ? 0 : (min as number))) % step);
```

Comment:
Nit/confusion: code wise I believe these operations work out (was concerned about how isNaN/Number coerces undefined/null). However, if min and max are required and typed as `number` then why is this all necessary?

Reply:
I get that it's confusing, however, they can be number or undefined according to our tests https://github.com/adobe/react-spectrum/blob/b39a2bbf308195b01e8369b19136e711b6b60867/packages/%40react-stately/utils/test/number.test.ts#L24

and unfortunately they aren't the last arg, so I can't make them 'optional' in the TS sense, but I can allow them to be undefined